### ISSUE TITLE
Fails to initialise with a HOT_WATER zone

### ISSUE THREAD

**Issue by thoukydides** [SeydX/homebridge-tado-platform#80]
Using a minimal `config.json`:
```
{
    "bridge":
    {
        "name":         "Homebridge Tado",
        "username":     "...",
        "port":         ...,
        "pin":          "..."
    },

    "platforms":
    [{
        "platform":     "TadoPlatform",
        "name":         "Tado",
        "debug":        false,
        "user": [{
            "username": "...",
            "password": "..."
        }]
    }]
}
```

attempting to start homebridge with this plugin results in the following error:
```
[Tado] An error occured during setting up plugin!
[Tado] TypeError: Cannot read property 'celsius' of undefined
    at Object.add (/usr/local/lib/node_modules/homebridge-tado-platform/src/tado/tado-config.js:126:45)
    at processTicksAndRejections (internal/process/task_queues.js:85:5)
    at TadoPlatform.setupPlugin (/usr/local/lib/node_modules/homebridge-tado-platform/src/platform.js:131:29)
```

Enabling debug reveals that the problem occurs when processing the `HOT_WATER` zone which does not support any temperature control, so there is no `capabilities.temperatures` object:
```
[Tado] Configuration: Checking access token..
[Tado] Configuration: Access token NOT expired
[Tado] Configuration: Using https://my.tado.com
[Tado] Configuration: API request GET /api/v2/homes/329647/zones <pending>
[Tado] Configuration: API request GET /api/v2/homes/329647/zones <success>
[Tado] Configuration: API request GET /api/v2/homes/329647/zones <response> [{"id":0,"name":"Hot Water","type":"HOT_WATER","dateCreated":"2019-05-16T15:57:55.867Z","deviceTypes":["BU01","RU02"],"devices":[{"deviceType":"BU01","serialNo":"BU2976583680","shortSerialNo":"BU2976583680","currentFwVersion":"81.1","connectionState":{"value":true,"timestamp":"2021-05-09T11:46:10.672Z"},"characteristics":{"capabilities":[]},"isDriverConfigured":true,"duties":["ZONE_DRIVER"]},{"deviceType":"RU02","serialNo":"RU2758741760","shortSerialNo":"RU2758741760","currentFwVersion":"82.1","connectionState":{"value":true,"timestamp":"2021-05-09T11:36:32.297Z"},"characteristics":{"capabilities":["INSIDE_TEMPERATURE_MEASUREMENT","IDENTIFY"]},"batteryState":"NORMAL","duties":["ZONE_UI","ZONE_LEADER"]}],"reportAvailable":false,"supportsDazzle":false,"dazzleEnabled":false,"dazzleMode":{"supported":false},"openWindowDetection":{"supported":false}},{"id":1,"name":"Living/Dining Room","type":"HEATING","dateCreated":"2019-05-07T15:14:31.395Z","deviceTypes":["RU02","VA02","VA02","VA02"],"devices":[{"deviceType":"RU02","serialNo":"RU...","shortSerialNo":"RU...","currentFwVersion":"82.1","connectionState":{"value":true,"timestamp":"2021-05-09T11:36:13.725Z"},"characteristics":{"capabilities":["INSIDE_TEMPERATURE_MEASUREMENT","IDENTIFY"]},"batteryState":"NORMAL","duties":["ZONE_UI","ZONE_LEADER"]},{"deviceType":"VA02","serialNo":"VA...","shortSerialNo":"VA...","currentFwVersion":"79.1","connectionState":{"value":true,"timestamp":"2021-05-09T11:46:03.776Z"},"characteristics":{"capabilities":["INSIDE_TEMPERATURE_MEASUREMENT","IDENTIFY"]},"mountingState":{"value":"CALIBRATED","timestamp":"2021-03-27T16:29:37.302Z"},"batteryState":"NORMAL","childLockEnabled":false,"duties":["ZONE_UI","ZONE_DRIVER"]},{"deviceType":"VA02","serialNo":"VA...","shortSerialNo":"VA...","currentFwVersion":"79.1","connectionState":{"value":true,"timestamp":"2021-05-09T11:43:39.333Z"},"characteristics":{"capabilities":["INSIDE_TEMPERATURE_MEASUREMENT","IDENTIFY"]},"mountingState":{"value":"CALIBRATED","timestamp":"2021-05-09T07:09:40.890Z"},"batteryState":"NORMAL","childLockEnabled":false,"duties":["ZONE_UI","ZONE_DRIVER"]},{"deviceType":"VA02","serialNo":"VA...","shortSerialNo":"VA...","currentFwVersion":"79.1","connectionState":{"value":true,"timestamp":"2021-05-09T11:48:00.600Z"},"characteristics":{"capabilities":["INSIDE_TEMPERATURE_MEASUREMENT","IDENTIFY"]},"mountingState":{"value":"CALIBRATED","timestamp":"2021-02-11T18:21:13.275Z"},"batteryState":"NORMAL","childLockEnabled":false,"duties":["ZONE_UI","ZONE_DRIVER"]}],"reportAvailable":false,"supportsDazzle":true,"dazzleEnabled":true,"dazzleMode":{"supported":true,"enabled":true},"openWindowDetection":{"supported":true,"enabled":true,"timeoutInSeconds":3600}},{"id":8,"name":"Hallway","type":"HEATING","dateCreated":"2019-05-16T15:57:52.838Z","deviceTypes":["RU02"],"devices":[{"deviceType":"RU02","serialNo":"RU...","shortSerialNo":"RU...","currentFwVersion":"82.1","connectionState":{"value":true,"timestamp":"2021-05-09T11:36:32.297Z"},"characteristics":{"capabilities":["INSIDE_TEMPERATURE_MEASUREMENT","IDENTIFY"]},"batteryState":"NORMAL","duties":["ZONE_UI","ZONE_LEADER"]}],"reportAvailable":false,"supportsDazzle":true,"dazzleEnabled":true,"dazzleMode":{"supported":true,"enabled":true},"openWindowDetection":{"supported":true,"enabled":true,"timeoutInSeconds":3600}},{"id":4,"name":"Office","type":"HEATING","dateCreated":"2019-05-10T15:58:22.057Z","deviceTypes":["VA02"],"devices":[{"deviceType":"VA02","serialNo":"VA...","shortSerialNo":"VA...","currentFwVersion":"79.1","connectionState":{"value":true,"timestamp":"2021-05-09T11:47:57.781Z"},"characteristics":{"capabilities":["INSIDE_TEMPERATURE_MEASUREMENT","IDENTIFY"]},"mountingState":{"value":"CALIBRATED","timestamp":"2021-02-11T23:58:52.912Z"},"batteryState":"NORMAL","childLockEnabled":false,"duties":["ZONE_UI","ZONE_DRIVER","ZONE_LEADER"]}],"reportAvailable":false,"supportsDazzle":true,"dazzleEnabled":true,"dazzleMode":{"supported":true,"enabled":true},"openWindowDetection":{"supported":true,"enabled":true,"timeoutInSeconds":3600}},{"id":3,"name":"Bedroom","type":"HEATING","dateCreated":"2019-05-07T15:21:23.807Z","deviceTypes":["RU02","VA02"],"devices":[{"deviceType":"RU02","serialNo":"RU...","shortSerialNo":"RU...","currentFwVersion":"82.1","connectionState":{"value":true,"timestamp":"2021-05-09T11:40:19.557Z"},"characteristics":{"capabilities":["INSIDE_TEMPERATURE_MEASUREMENT","IDENTIFY"]},"batteryState":"NORMAL","duties":["ZONE_UI","ZONE_LEADER"]},{"deviceType":"VA02","serialNo":"VA...","shortSerialNo":"VA...","currentFwVersion":"79.1","connectionState":{"value":true,"timestamp":"2021-05-09T11:37:51.220Z"},"characteristics":{"capabilities":["INSIDE_TEMPERATURE_MEASUREMENT","IDENTIFY"]},"mountingState":{"value":"CALIBRATED","timestamp":"2021-02-11T19:51:02.414Z"},"batteryState":"NORMAL","childLockEnabled":false,"duties":["ZONE_UI","ZONE_DRIVER"]}],"reportAvailable":false,"supportsDazzle":true,"dazzleEnabled":true,"dazzleMode":{"supported":true,"enabled":true},"openWindowDetection":{"supported":true,"enabled":true,"timeoutInSeconds":3600}},{"id":6,"name":"Guest Bedroom","type":"HEATING","dateCreated":"2019-05-10T16:07:42.548Z","deviceTypes":["VA02"],"devices":[{"deviceType":"VA02","serialNo":"VA...","shortSerialNo":"VA...","currentFwVersion":"79.1","connectionState":{"value":true,"timestamp":"2021-05-09T11:48:10.823Z"},"characteristics":{"capabilities":["INSIDE_TEMPERATURE_MEASUREMENT","IDENTIFY"]},"mountingState":{"value":"CALIBRATED","timestamp":"2021-02-12T00:31:05.842Z"},"batteryState":"NORMAL","childLockEnabled":false,"duties":["ZONE_UI","ZONE_DRIVER","ZONE_LEADER"]}],"reportAvailable":false,"supportsDazzle":true,"dazzleEnabled":true,"dazzleMode":{"supported":true,"enabled":true},"openWindowDetection":{"supported":true,"enabled":true,"timeoutInSeconds":3600}},{"id":5,"name":"Bathroom","type":"HEATING","dateCreated":"2019-05-10T16:03:53.514Z","deviceTypes":["VA02"],"devices":[{"deviceType":"VA02","serialNo":"VA...","shortSerialNo":"VA...","currentFwVersion":"79.1","connectionState":{"value":true,"timestamp":"2021-05-09T11:44:30.963Z"},"characteristics":{"capabilities":["INSIDE_TEMPERATURE_MEASUREMENT","IDENTIFY"]},"mountingState":{"value":"CALIBRATED","timestamp":"2021-02-02T14:15:22.128Z"},"batteryState":"NORMAL","childLockEnabled":false,"duties":["ZONE_UI","ZONE_DRIVER","ZONE_LEADER"]}],"reportAvailable":false,"supportsDazzle":true,"dazzleEnabled":true,"dazzleMode":{"supported":true,"enabled":true},"openWindowDetection":{"supported":true,"enabled":true,"timeoutInSeconds":3600}},{"id":7,"name":"Spare Room","type":"HEATING","dateCreated":"2019-05-10T16:12:43.121Z","deviceTypes":["VA02"],"devices":[{"deviceType":"VA02","serialNo":"VA...","shortSerialNo":"VA...","currentFwVersion":"79.1","connectionState":{"value":true,"timestamp":"2021-05-09T11:47:11.975Z"},"characteristics":{"capabilities":["INSIDE_TEMPERATURE_MEASUREMENT","IDENTIFY"]},"mountingState":{"value":"CALIBRATED","timestamp":"2021-02-02T14:22:08.102Z"},"batteryState":"NORMAL","childLockEnabled":false,"duties":["ZONE_UI","ZONE_DRIVER","ZONE_LEADER"]}],"reportAvailable":false,"supportsDazzle":true,"dazzleEnabled":true,"dazzleMode":{"supported":true,"enabled":true},"openWindowDetection":{"supported":true,"enabled":true,"timeoutInSeconds":3600}}]
[Tado] Configuration: Checking access token..
[Tado] Configuration: Access token NOT expired
[Tado] Configuration: Using https://my.tado.com
[Tado] Configuration: API request GET /api/v2/homes/329647/zones/0/capabilities <pending>
[Tado] Configuration: API request GET /api/v2/homes/329647/zones/0/capabilities <success>
[Tado] Configuration: API request GET /api/v2/homes/329647/zones/0/capabilities <response> {"type":"HOT_WATER","canSetTemperature":false}
[Tado] An error occured during setting up plugin!
[Tado] TypeError: Cannot read property 'celsius' of undefined
     at Object.add (/usr/local/lib/node_modules/homebridge-tado-platform/src/tado/tado-config.js:126:45)
     at processTicksAndRejections (internal/process/task_queues.js:85:5)
     at TadoPlatform.setupPlugin (/usr/local/lib/node_modules/homebridge-tado-platform/src/platform.js:131:29)
```

This is with:
* homebridge-tado-platform 6.0.12
* homebridge 1.3.1
* node 12.8.0

**Comment by SeydX** [comment]
Fixed in v6.0.13

**Comment by thoukydides** [comment]
I confirm that this issue is fixed in version 6.0.13.
Thank you @SeydX.
